test/js/pageobjects/Welcome.js

```diff
@@ -44,6 +44,14 @@ class Welcome extends Base {
   async closeModal() {
     this.closeButton.click()
   }
+
+  async backArrowNotDisplayed() {
```

Comment:
I think the naming here is a bit misleading. A function name should say what it does, e.g. `checkBackArrowIsDisplayed`

Reply:
I'm expecting it not to be displayed hence I catch the error. Selenium doesn't provide the check to verify whether the element is not displayed.

Improved the name of the function 561ae1f